config/routes.rb

```diff
@@ -2,6 +2,7 @@
 
 Rails.application.routes.draw do
   devise_for :users, :controllers => { :sessions => "sessions" }
+  get '/users/team_members', to: 'users#team_members', as: 'team'
```

Comment:
Is there a path that makes more sense than `/users/team_members` for this route? Using `as` suggests you think `/team` makes more sense. How could you rewrite this without the `as` to achieve that?

Reply:
The path was changed to /team